internal/slicer/report.go

```diff
@@ -0,0 +1,64 @@
+package slicer
+
+import (
+	"fmt"
+	"io/fs"
+	"path/filepath"
+	"strings"
+
+	"github.com/canonical/chisel/internal/fsutil"
+	"github.com/canonical/chisel/internal/setup"
+)
+
+type ReportEntry struct {
```

Comment:
Just a quick item for a follow up: this looks a lot like fsutil.Info.  Maybe we could call the latter as fsutil.Entry or something analogous for both, and bring them closer together. Let's do a brief brainstorm on our next call.

Reply:
True. the difference is that `fsutil.Info` will only hold information about the file being created like that hash, size and name while `ReportEntry` can hold additional information that only makes sense in the context of the Report. Right now the slices that reported the file but we could also add fields to see if a file was deleted, if it is mutable or marked with until, etc. That is why I created two different structs, for the separation of concerns.